pkg/netstate/netstate.go

```diff
@@ -83,6 +109,18 @@ func (s *Server) List(ctx context.Context, req *pb.ListRequest) (*pb.ListRespons
 	s.logger.Debug("entering netstate list")
 
 	pathKeys, err := s.DB.List()
+	
+	xAPIKeyBytes := []byte(req.XApiKey)
+	if err := validateAuth(xAPIKeyBytes); err != nil {
+		c := []byte("Unauthorized Request")
+		s.logger.Error("unauthorized request")
+
+		return &pb.ListResponse{
+			Filepaths: [][]byte{c},
```

Comment:
Oh just realized `Filepaths` needs to be changed to `Paths`

Reply:
changed already :) i caught that :)